apps/api/src/index.ts

```diff
@@ -18,8 +16,10 @@ export default {
   async fetch(request: Request, env: APIEnv): Promise<Response> {
     const req = request.clone();
     const { url, method, headers } = request;
+
     const path = new URL(url).pathname;
-    if (method !== 'GET' && method !== 'OPTIONS') {
+
+    if (method !== 'GET' && method !== 'OPTIONS' && method !== 'POST') {
```

Comment:
Doesn't this allow someone to POST to any endpoint?

Reply:
No, just the accounts endpoint, I'm looking at how I can lock it down